.flowconfig

```diff
@@ -13,6 +13,7 @@
 
 ; Ignore unexpected extra "@providesModule"
 .*/node_modules/.*/node_modules/fbjs/.*
+.*/node_modules/react-dom/.*
```

Comment:
Not sure why this didn't cause issues before these changes but we have `react-dom` as a dev dep (for tests I guess) so just ignore it to avoid other haste conflicts.